das/checkpoint.go

```diff
@@ -25,6 +26,7 @@ func newCheckpoint(stats SamplingStats) checkpoint {
 		workers = append(workers, workerCheckpoint{
 			From: w.Curr,
 			To:   w.To,
+			Kind: w.Kind,
```

Comment:
[proposal][non-blocking]: can we have a dedicated issue to keep this awesome [ADR-012](https://github.com/celestiaorg/celestia-node/blob/main/docs/adr/adr-012-daser-parallelization.md) up to date? 🙏

Reply:
Good suggestion, thank you! I've opened a tracking issue https://github.com/celestiaorg/celestia-node/issues/1886